ocaml/doc/wire-protocol.md

```diff
@@ -56,7 +56,7 @@ These types are mapped onto XML-RPC types in a straight-forward manner:
   `(k -> v) map` type is only valid when `k` is a `string`, `ref`, or
   `int`, and in each case the keys of the maps are stringified as
   above. For example, the `(string -> double) map` containing the mappings
-  _Mike &rarr; 2.3_ and _John &rarr; 1.2_ would be represented as:
+  _Mike -> 2.3_ and _John -> 1.2_ would be represented as:
```

Comment:
Ok, it seems the output differs per system... No problem. If  I remember well there are some more instances of `&rarr;` in the markdown_backend.ml you may want to change.